pkgs/by-name/zs/zsh-histdb/package.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, stdenv, fetchFromGitHub, ...}:
+
+stdenv.mkDerivation {
+  pname = "zsh-histdb";
+  version = "unstable-2022-01-18";
+
+  src = fetchFromGitHub {
+    owner = "larkery";
+    repo = "zsh-histdb";
+    rev = "30797f0c50c31c8d8de32386970c5d480e5ab35d";
+    hash = "sha256-PQIFF8kz+baqmZWiSr+wc4EleZ/KD8Y+lxW2NT35/bg=";
+  };
+
+  installPhase = ''
```

Comment:
> You will need sqlite3 and the usual coreutils commands installed on your PATH

Shall we wrap/fixup the scripts?

Reply:
I'm not sure,  I used to add `buildInputs = [ pkgs.zsh ];`,  but considering nobody will use this package without zsh, so, this seems not make sense...

Reply:
I mean wrapping with sqlite3 (and other commands these scripts need)